cmplib/cmpcl_read.c

```diff
@@ -55,6 +57,12 @@ static void zeroize( void *v, size_t n ) {
 
 #define PARSE_ASN1_CONSTRUCTED_TAG(expected) ((ret = mbedtls_asn1_get_tag( &p, end, &len, MBEDTLS_ASN1_CONSTRUCTED | expected ) ) != 0)
 
+#define MBEDTLS_ERR(ret, err_buf, CMPCL_ER_BUF_LEN)  \
+			do{\
```

Comment:
Hmm, moving the decl to file-level was not what I meant - this might cause trouble with multi-threaded invocation.
Please move the decl inside the `while` loop.